_posts/2022-08-24-kubeflow-1.6-release.md

```diff
@@ -0,0 +1,146 @@
+ Predictor Spec, which provides a new way to specify models in KServe 0.8. Additionally, 1.6 cleans up a few CVEs in
+ the central dashboard and enables PodDefaults webhook to pick new certificates. In Notebooks, 1.6 adds the ability to
+ show objects from all namespaces and create a separate workflow for each notebook server. The Manifest Working Group
+ developed and tested each Kubeflow’s component manifest for Kubernetes 1.22, coupled with the manifests that can be
+ applied, the Pods of components that can become ready, and the basic objects that can be created.
+ 
+In parallel to developing these new software features, the Kubeflow Community also completed its annual user survey.
+ The survey generated many good user insights into requirements for Kubeflow, and you can see more on the
+ [survey results](https://docs.google.com/document/d/1TLl8iFWQ-x2pVAHV9G8GFYMgAOuN2VFOnZV2JQNA-uc/edit#). Of the many
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
+ to explain Kubeflow’s current functionality and build into the users’ new monitoring requirements. In addition,
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the 1.6 highlights in more detail.
+
+## Kubeflow v1.6 details
+
+From a new feature introduction standpoint, Kubeflow v1.6 includes an experimental release of the KFP v2’s new front-end,
+ back-end and SDK, which deliver a modern UI and DAG, first class support for metadata, and a simplified component
+ authoring experience. This slide deck, [Understand KFP v2](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_20),
+ provides a good overview of v2. 
+
+<br />
+
+The KFP v2 alpha introduces: 
+
+- An Argo-agnostic approach for creating and running pipelines 
+- A brand new DAG visualization, which uses the Pipeline Template and MLMD in this [pipeline spec](https://github.com/kubeflow/pipelines/blob/master/api/v2alpha1/pipeline_spec.proto)
+- Ability to set pipeline root for the root path of object storage when building pipeline template, and when creating pipeline run.
+- PIP install functionality from private PyPI repository [design](https://docs.google.com/document/d/1r7XKNncJ6du1EnX3iiUqOJXMMA2nvoXXUarOXe_sm1o/edit?resourcekey=0-pGC-6oMc2Iog5hKW1eJHAA#heading=h.x9snb54sjlu9)
```

Comment:
The pipelines WG says that existing lines 52 and line 53 should be deleted and replace with this line...

[Streamlined Component Authoring] (https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)

Reply:
Done